autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/LoadAlertConfigurationRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.sequenceiq.periscope.api.model;
+
+import javax.validation.Valid;
+
+import com.sequenceiq.periscope.doc.ApiDescription.LoadAlertJsonProperties;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+public class LoadAlertConfigurationRequest extends AbstractAlertJson {
+
+    @ApiModelProperty(LoadAlertJsonProperties.LOAD_ALERT_CONFIGURATION_MIN_RESOUCE_VALUE)
```

Comment:
ScalingConfigurationRequest is at cluster level, and this can be overriding value for the alert. If not this can be removed and the cluster level value be used.

Reply:
I would say remove ScalingConfigurationRequest rather than this.
Actually, ScalingConfigurationRequest could be changed to allow users to configure aspects like
- If scaling up, max number of nodes to add in one 'scale-up' event. (This is hardcoded to 100 in CB at the moment.

Anyway, I'd remove it for now.